config/sync/field.field.node.recipe.field_r_categories.yml

```diff
@@ -11,8 +11,8 @@ field_name: field_r_categories
 entity_type: node
 bundle: recipe
 label: Categories
-description: ''
-required: false
+description: 'Categorise your recipe to make it easier for other users to find. You can categorise it based on its main protein, the meal type, or create your own category. '
```

Comment:
Feels too much, you just wanna say if you can't find something to match your category you can create your own. Something along those lines

Reply:
done